### PR TITLE
Fix: Entering appointment infos on safari

### PR DESCRIPTION
**Pull request by LomyW**
### Description
Select element takes focus from input element on react native web on Safari browser

### Done
- [x] Added a select specific for safari

### Tested
- [x] Safari
- [x] Firefox
- [x] Chrome

### CODE REVIEW

**Comment by timidak** [comment]
@LomyW is this branch up to date with main? 
Seems like the checks are failing, because of the graphql validation.

**Comment by LomyW** [comment]
> @LomyW is this branch up to date with main? Seems like the checks are failing, because of the graphql validation.

yes it is up to date with main.

**Review by dhenkel92 — DISMISSED** [review]

File: src/widgets/SafariSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Select as NBSelect } from 'native-base';
+
+const SafariSelect = (props: any) => {
```
Comment (timidak) [inline src/widgets/SafariSelect.tsx:3]: `props` shouldn't be typed as any. Use the props the `NBSelect` uses.
Also, `SafariSelect` seems like it's only used for Safari?

File: src/widgets/SafariSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    const isSafari = () => /Safari/.test(navigator.userAgent) && /Apple Computer/.test(navigator.vendor);
+
+    return <NBSelect {...props} selection={isSafari() ? 1 : null} />;
+};
+
+SafariSelect.Item = NBSelect.Item;
+export default SafariSelect;
+export { SafariSelect };
```
Comment (timidak) [inline src/widgets/SafariSelect.tsx:11]: Why do we have to export statements? I'd say either try the named export or the default one.

File: src/widgets/SafariSelect.tsx (hunk 1 above)
Comment (timidak) [reply src/widgets/SafariSelect.tsx:3]: I would add some additional documentation, why we had do build this select, maybe with a link to the issues and a short description of the problem as JS/TS-Doc.

**Review by timidak — CHANGES_REQUESTED** [review]
Just a few general things, could you please test it again on every browser, after you implemented the changes.

**Review by timidak — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**